### PR TITLE
[W6.6h][W11-B1] Li JiaYao

### PR DESCRIPTION
**Pull request by JYL123**
Add use case: rename tag

### CODE REVIEW

File: doc/DeveloperGuide.md [hunk 1]
```diff
@@ -87,6 +87,42 @@ Use case ends.
...
+
+All process working fine. 
+
+**Guarantees**
+
+The tag is renamed.
+
+**MSS**
+
+1. User enters command 'rename tag'
```
Comment (okkhoy) [inline doc/DeveloperGuide.md]: no need to specify the command here. command you provide is implementation specific.

File: doc/DeveloperGuide.md [hunk 2]
```diff
@@ -87,6 +87,42 @@ Use case ends.
...
+2. AddressBook shows a list of tags ordered with index
+3. User enters the index of the tag that is needed to be renamed, and the new tag name with a space in between
+4. AddressBook displays the message that the tag is renamed successfully <br>
+Use case ends.
+
+**Extensions**
+
+2a. The list is empty
```
Comment (okkhoy) [inline doc/DeveloperGuide.md]: In use case extensions, the numbering is use to describe scenarios that can happen, preventing the user from going to the next step.
so, this is better off being 1a.

**Comment by okkhoy** [comment]
@JYL123 
Some comments added. Please ack & close the PR after reading comments.

**Comment by JYL123** [comment]
Thanks for reviewing. Changes are made. PR is closed now.

**Pull request closed without merging**